specs/parameters/id-parameter.yml

```diff
@@ -0,0 +1,6 @@
+in: path
+name: "id"
+schema:
+  type: string
+required: true
+description: Identifier for entities
```

Comment:
Duplicate - there is already an existing common-id-parameter.yml. We can stick with that or we can refactor that for this name.

Reply:
This was created in order to be generic (as it has the same type, scope and everything)